Dockerfile

```diff
@@ -6,7 +6,6 @@ WORKDIR /app
 ARG DISABLE_EXTRAS=false
 
 COPY pyproject.toml ./
-COPY poetry.lock ./
```

Comment:
Why did you remove it? It's needed for the environment reproducibility

Reply:
I removed it to test if CI is able to build the image. It was failing on the writing lock file part. It seems to work now that I have updated the lock file with `poetry lock --no-update`.